modules/ossm-about-adding-projects-using-smm.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/ossm-create-mesh.adoc
+
+:_content-type: CONCEPT
+[id="ossm-about-adding-projects-using-smm_{context}"]
+= About adding projects using the member resource
+
+A `ServiceMeshMember` resource provides a way to add a project to a service mesh without modifying the `ServiceMeshMemberRoll` resource. To add a project, create a `ServiceMeshMember` resource in the project that you want to add to the service mesh. When the {SMProductShortName} Operator processes the `ServiceMeshMember` object, the project appears in the `status.members` list of the `ServiceMeshMemberRoll` resource. Then, the services that reside in the project are made available to the mesh. For more information, see the instructions for adding projects to the service mesh using the `ServiceMeshMember` resource with the xref:../../service_mesh/v2x/ossm-create-mesh.adoc#ossm-adding-project-using-smm-resource-console_ossm-create-mesh[web console] or with the xref:../../service_mesh/v2x/ossm-create-mesh.adoc#ossm-adding-project-using-smm-resource-cli_ossm-create-mesh[CLI].
```

Comment:
I think we should say here that `ServiceMeshMemberRoll` needs to be created before you start to add `ServiceMeshMember`, something like this:

```
A `ServiceMeshMember` resource provides a way to add a project to a service mesh without modifying the `ServiceMeshMemberRoll` resource. First you will need to create a `ServiceMeshMemberRoll` (You can follow this instructions (add the link)), as soon the ServiceMeshMemberRoll` is created to add a project, create a `ServiceMeshMember` resource in the project that you want to add to the service mesh. When the {SMProductShortName} Operator processes the `ServiceMeshMember` object, the project appears in the `status.members` list of the `ServiceMeshMemberRoll` resource. Then, the services that reside in the project are made available to the mesh. For more information, see the instructions for adding projects to the service mesh using the `ServiceMeshMember`

wdyt?
```

Reply:
@fjglira this is a conceptual topic that explains what the SMM is and why someone might use it. The prerequisites section of the procedure for adding a project using the SMM states you need the name of the ServiceMeshControlPlane resource and the name of the project that the resource belongs to. So I think we're good on this.

Reply:
Understood